platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/AbstractStsRealm.java

```diff
@@ -110,6 +113,10 @@
 
     private Boolean useKey = null;
 
+    private Cache<Element, SecurityToken> cache = CacheBuilder.newBuilder()
+            .expireAfterWrite(1, TimeUnit.MINUTES)
```

Comment:
Should this cache be slightly less than one minute to help prevent any possible race conditions then if one *were* to set it at a minute? (I understand here this is just a possibility, not the norm)